test/service.spec.js

```diff
@@ -43,6 +43,16 @@
         expect(service.config.sslEnabled).to.equal(true);
         return expect(service.config.maxRetries).to.equal(5);
       });
+      it('should inherit the config from global config if it is not set specificly', function() {
```

Comment:
Make sure to put an empty line between tests.

Reply:
Seems like there's no new line in any test files. Probably because they are generated from other scripts previously.